working-groups/wg-tidb-distributions.md

```diff
@@ -0,0 +1,34 @@
+# TiDB Distributions Working Group
+
+## Goals
+
+This working group aims to build distribution packages of TiDB for different platforms and maintain them for the long-term supports.
+
+## Meetings
+
+* To be added.
```

Comment:
We can add a simple statement to explain why there is not a meeting, or just remove this section.

Reply:
I removed this section.